KomaMRICore/src/KomaMRICore.jl

```diff
@@ -43,19 +40,15 @@ include("simulation/GPUFunctions.jl")
 # include("simulation/other/OffResonanceModel.jl")
 include("simulation/TrapezoidalIntegration.jl")
 include("simulation/SimulatorCore.jl")
-include("io/ISMRMRD.jl")
 
 #Main
 export γ #gyro-magnetic ratio [Hz/T]
 export Scanner, Sequence, Phantom
 export Grad, RF, ADC, Delay
 export Mag, dur
-#Pulseq
-export read_seq
-#ISMRMRD
-export signal_to_raw_data
+export recursive_merge
```

Comment:
why are you exporting this?

Reply:
I didn't intend to make any changes to KomaMRICore. This was a mistake in the commit.